Packs/Core/Playbooks/playbook-Exchange_forwarding_rule_configured.yml

```diff
@@ -0,0 +1,1797 @@
+id: Exchange forwarding rule configured
+version: -1
+name: Exchange forwarding rule configured
+description: "This playbook addresses the following alerts:\n\n- External Exchange
+  inbox forwarding rule configured.\n- Suspicious Exchange inbox forwarding rule configured.\n-
+  Suspicious Exchange email-hiding inbox rule.\n\nPlaybook Stages:\n \nTriage: \n\n-
+  The playbook retrieves the caller's IP, the forwarding email address, and the domain.\n\nEarly
+  Containment:\n\n- The playbook checks if the IP or domain of the forwarding email
+  address is malicious. If so, it suggests blocking the IP using PAN-OS while continuing
+  the investigation in parallel.\n\nInvestigation:\n\n- The playbook checks for suspicious
+  behaviors, including whether an Exchange admin created the rule outside of working
+  hours, from an unusual geolocation, or if the user who created the rule has a high-risk
+  score. It then aggregates all evidence collected during the investigation.\n\nContainment:\n\n-
+  If at least two suspicious evidence are found, the playbook executes soft response
+  actions, including signing the user out and deleting the forwarding email address
+  from the user account mailbox. The user will be notified of these actions via email.\n-
+  If two or more pieces of suspicious evidence are found, the playbook will initiate
```

Comment:
This descriptions does not mention the hard remediation and the action we take there. please fix